app/lib/import_export.rb

```diff
@@ -144,10 +142,12 @@ module ImportExport # rubocop:todo Metrics/ModuleLength
     WY: 'Wyoming'
   }.freeze
 
+  VALID_RACES = {default: ['2106-3', '2054-5', '1002-5', '2028-9', '2076-8'], exclusive: ['UNK', 'OTH', 'ASKU']}
```

Comment:
I'm not crazy about the default races being these codes. It makes things a lot less readable as opposed to having the words these number codes represent. What are the other options? 

cc: @holmesie

Reply:
I'm not crazy about it either, to be honest: the reason I went with these is because when converting from a FHIR object the codes are a single point of truth. With some extra time, I could probably make some constant arrays and parsing such that we don't store them as codes.

Reply:
Hopping in since this could change some of the validation I've been doing, but I think when we ingest a FHIR Patient that has ethnicity codes set, we convert the coded value to something human readable (like "Hispanic or Latino"). So would a similar approach work with the race codes? They seem like similar situations that we might expect to work the same.

Reply:
@ngfreiter I like that approach